src/react-query/queryClient.ts

```diff
@@ -13,8 +14,25 @@ export const queryClient = new QueryClient({
   },
 });
 
+
+const storage = new MMKV();
+
+const clientStorage = {
```

Comment:
would love to use a legacyStorage here like im doing for zustand so everything id in thee same place

Reply:
this should not be using mmkv directly